docs/articles/verifying-signatures.md

```diff
@@ -8,54 +38,56 @@ To verify the validity of a signature for an image, zot makes use of two types o
 
 - A certificate file that is used to sign an image with `notation`
 
-These files can be uploaded by the user using the `mgmt` extension of the zot API, as shown in the following examples:
+Upload these files using an extension of the zot API, as shown in the following examples:
 
-1. **To upload a public key**: 
+- **To upload a public key for cosign**: 
 
     *API path*
     ```
-    /v2/_zot/ext/mgmt?resource=signatures&tool=cosign
+    /v2/_zot/ext/cosign"
     ```
     *Example request*
     ```
-    curl --data-binary @file.pub -X POST "http://localhost:8080/v2/_zot/ext/mgmt?resource=signatures&tool=cosign"
+    curl --data-binary @file.pub -X POST "http://localhost:8080/v2/_zot/ext/cosign"
     ```
+    *Result*
 
-2. **To upload a certificate**:
+    The uploaded file is stored in the `_cosign` directory under the `rootDir` specified in the zot configuration file.
+
+- **To upload a certificate for notation**:
 
     *API path*
     ```
-    /v2/_zot/ext/mgmt?resource=signatures&tool=notation&truststoreType=ca&truststoreName=upload-cert
-    ```
-    *Example request*
-    ```
-    curl --data-binary @filet.crt -X POST "http://localhost:8080/v2/_zot/ext/mgmt?resource=signatures&tool=notation&truststoreType=ca&truststoreName=upload-cert"
+    /v2/_zot/ext/notation?truststoreType=ca&truststoreName=upload-cert
```

Comment:
@mbshields the query parameters have been changed. Please see the updated pkg/extensions/README_imagetrust.md in https://github.com/project-zot/zot/pull/1744

Reply:
changed query parameters, removing truststoreName, etc

Reply:
@Andreea-Lupu please review.

Otherwise looks good from my side.